### PR TITLE
Tweaks to build against gcc-10.

### PR DESCRIPTION
**Pull request by trofi**

### CODE REVIEW

**Comment by JosJuice** [comment]
@dolphin-emu-bot rebuild

File: Source/Core/Common/StringUtil.h [hunk 1]
```diff
@@ -8,6 +8,7 @@
 #include <cstddef>
 #include <cstdlib>
 #include <iomanip>
+#include <limits> /* std::numeric_limits */
```
Comment (jordan-woyak) [inline Source/Core/Common/StringUtil.h]: I think these comments are unnecessary.
We don't do this anywhere else and no one is going to keep them updated as header requirements change.

Comment (JosJuice) [reply Source/Core/Common/StringUtil.h]: Agreed. This pull request LGTM if the comments are removed.

Comment (trofi) [reply Source/Core/Common/StringUtil.h]: Sounds good. Re-pushed patches without a comment.

**Comment by JosJuice** [comment]
@dolphin-emu-bot rebuild

**Pull request merged**